### PR TITLE
New Package: spotify-qt 

### PR DESCRIPTION
**Pull request by T-Dynamos**
This is my first package on termux

### CODE REVIEW

File: tur/spotify-qt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/kraxarn/spotify-qt"
+TERMUX_PKG_DESCRIPTION="Lightweight Spotify client using Qt"
+TERMUX_PKG_LICENSE="GNU General Public License v3.0"
+TERMUX_PKG_MAINTAINER="@termux"
```
Comment (kcubeterm) [inline tur/spotify-qt]: either put @tur or your own username.

**Comment by T-Dynamos** [comment]
I added qt5-base but it gave error 
```
ERROR: Package spotify-qt depends on non-existing package "qt5-qtbase"
```
When i removed it
```
  Could not find a package configuration file provided by "Qt5" with any of
  the following names:
    Qt5Config.cmake
    qt5-config.cmake
```

File: tur/spotify-qt (hunk 1 above)
Comment (T-Dynamos) [reply tur/spotify-qt]: Ok

**Comment by kcubeterm** [comment]
need to add x11-packages in repo.json  as well. I am working on it.

File: tur/spotify-qt/build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/kraxarn/spotify-qt"
+TERMUX_PKG_DESCRIPTION="Lightweight Spotify client using Qt"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@Tur"
```
Comment (licy183) [inline tur/spotify-qt/build.sh]: I'd prefer `@termux-user-repository`. Actually, `@Tur` is a github user.

File: tur/spotify-qt/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+TERMUX_PKG_VERSION="3.8"
+TERMUX_PKG_SRCURL="https://github.com/kraxarn/spotify-qt/archive/refs/tags/v3.8.tar.gz"
```
Comment (licy183) [inline tur/spotify-qt/build.sh:6]: Please replace `3.8` with `$TERMUX_PKG_VERSION`.

File: tur/spotify-qt/build.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+TERMUX_PKG_SHA256="deb12800b9fb4f501f8dfc97a2fdc4989849fe8bbf6f38d79503214e03afe5ad"
+TERMUX_PKG_DEPENDS="qt5-qtbase, zlib"
```
Comment (licy183) [inline tur/spotify-qt/build.sh]: It seems to depend on `qt5-qtsvg` too.

**Comment by 2096779623** [comment]
https://github.com/termux/termux-packages/issues

**Pull request closed without merging**